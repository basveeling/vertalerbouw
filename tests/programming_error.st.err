// Assign a char to an int variable

var int foo;

foo := 'a'; 

print(foo);
