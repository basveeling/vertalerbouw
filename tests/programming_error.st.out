ERROR: recognition exception thrown by compiler: := (5:4) operand type does not match identifier type